src/utils/conditionals.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash'
+
+export function pathFinder (valueObj, prevPath) {
+  return function (path) {
+    if (!Array.isArray(path)) {
+      path = path.split('.').reverse()
+    }
+    let nextInPath = _.last(path)
+
+    if (nextInPath === '') { // . for sibling
```

Comment:
Don't understand this comment or the one on line 12. What is the significance of the `.` and `..`?